acct_mgt_test.py

```diff
@@ -75,75 +85,60 @@ def wait_while(project, pod_name, statuses, time_out=300):
 
 
 def wait_until_done(oc_cmd, finished_pattern, time_out=30, decrement=5):
-    p1 = re.compile(finished_pattern)
+    """
+    This wait while an oc command is running, looking for a pattern that
+    indicates it is finished.
+    """
+    pattern1 = re.compile(finished_pattern)
```

Comment:
can we call this `pattern` since there's no `pattern2` here?

Reply:
Sure, why not